finder-app/finder.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#Author:Isha Sharma
+
+#check input params, lt-less than
+if [ $# -lt 2 ]; then
+    echo "Error : 2 required aprameters, not specified"
+    exit 1
```

Comment:
@ishassharmaa I like the idea to print an error message here but how could you make the error more helpful for the user?